### PR TITLE
erc1155 tx struct and ERC1155Transfers method

### PR DESCRIPTION
**Pull request by Pashteto**

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -1,5 +1,7 @@
 module github.com/nanmu42/etherscan-api
 
-go 1.13
+go 1.19
```
Comment (nanmu42) [inline go.mod:3]: Since we are not using latest go features, keeping the go version old would be better for our existing users.

File: go.mod [hunk 2]
```diff
@@ -1,5 +1,7 @@
...
 
-require github.com/google/go-cmp v0.5.7 // indirect
+require github.com/google/go-cmp v0.5.7
+
+replace github.com/nanmu42/etherscan-api v1.9.0 => github.com/Pashteto/etherscan-api v1.0.0
```
Comment (nanmu42) [inline go.mod:7]: This replace command does not look right...

**Review by nanmu42 — COMMENTED** [review]
Thank you very much for your contribution!

There are few changes needed before I can proceed.

File: go.mod (hunk 2 above)
Comment (Pashteto) [reply go.mod]: i don't want to add the replace line into your main repo. I just wanted to get a link to the forked, edited package into my project and unfortunately it wasn't working.
I can revert to the commit cf99fda6d3e7b8c40e2f2e93ed29499c0fed52af if you want to merge these changes.

**Review by nanmu42 — APPROVED** [review]
Thank you for your contribution!

**Pull request merged**